dashboard/src/app/index.module.js

```diff
@@ -96,8 +96,51 @@ initModule.factory('AuthInterceptor', ($window, $cookies, $q, $location, $log) =
   };
 });
 
-initModule.config(['$routeProvider', '$locationProvider', '$httpProvider', ($routeProvider, $locationProvider, $httpProvider) => {
+initModule.factory('AddMachineTokenToUrlInterceptor', ($injector, $q) => {
```

Comment:
we've a reference there
https://github.com/eclipse/che/tree/master/dashboard#angularjs-recommandation

All injection required will be done through the constructor by adding also the @ngInject annotation.